### PR TITLE
[#4]: Add option to disable looping & german locale

### PR DESCRIPTION
**Pull request by fochlac**

### CODE REVIEW

File: js/player.js [hunk 1]
```diff
@@ -33,13 +35,51 @@ class YTShortsPlayer {
...
+  disableLoop(videoElement) {
+    setTimeout(() => {
+      videoElement.loop = false;
+    }, 1500);
+  }
+
+  disableLoopingIfNeeded(video) {
+    if (!this.disableInfiniteLoop) return;
+    if (video.isObserved) return;
+    video.isObserved = true;
```
Comment (EdoanR) [inline js/player.js:55]: Hey thanks for the PR! Unfortunately for me the infinite loop option just works when the page loads, but when toggling the looping option while the video is already loaded it does not change the infinite loop.

for example:
1. video has infinite loop.
2. disable infinite loop
3. video **still** have infinite loop
4. reloads the page, now the video does not have infinite loop
5. enable infinite loop
6. the video **stills** does not have infinite loop.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "controls-for-yt-shorts",
   "scripts": {
     "dev": "npm-run-all --parallel scss web-ext",
-    "web-ext": "node .ext.mjs",
+    "web-ext": "node .ext-template.mjs",
```
Comment (EdoanR) [inline package.json:5]: The `.ext-template.mjs` file was meant to be used as a template for the `.ext-mjs`, where the developer rename the file `.ext-template.mjs` to `.ext.mjs` (my bad I should made it clear). 

As the `.ext-mjs` is ignored by git, every developer can have a different testing environment for the extension without the risk pushing changes to it and overwriting the one of other developers. 

```suggestion
    "web-ext": "node .ext.mjs",
```

**Review by EdoanR — COMMENTED** [review]
The german locale and popup changes are great. Thanks a lot!

But I had some issues related to the disable loop feature, at least on my end. Please let me know if this happens to you too.

File: package.json (hunk 2 above)
Comment (fochlac) [reply package.json]: Might make sense to add a postinstall step that creates the file automatically if it doesnt exist.

File: js/player.js (hunk 1 above)
Comment (fochlac) [reply js/player.js:48]: Hmm, I set it up to switch on/off after a full page reload. I guess binding into the applyChanges was not too hard, now it switches instantly.

File: package.json (hunk 2 above)
Comment (fochlac) [reply package.json]: I just added that as well.

**Comment by EdoanR** [comment]
Seems all good, thank you! 😀

**Pull request merged**